server/formatter.js

```diff
@@ -113,11 +113,22 @@ function formatCode(html) {
     return `<code>${formatCodeContent(content)}</code>`
   })
 
-  html = html.replace(/&lt;%-(.+)%&gt;/g, (match, content) => {
-    if (!allowInlineCode) return '' // strip out scripts entirely when not permitted
-    const html = unescape(content)
-    return formatCodeContent(html)
-  })
+  // for inline code option
+  if (allowInlineCode) {
+    const matches = []
+    // get all code matches, push any that are not <pre> wrapped
+    html.replace(/&lt;%-.*?\s?%&gt;(.*?<\/pre>)?/g, (codeContent, closingPre) => {
+      if (!closingPre) matches.push(codeContent)
+    })
```

Comment:
I'm sure there is a better way to do this with something like `.filter` instead of abusing the replace callback, but need to think on it a bit more...

Reply:
Hey @afischer, I'm OK with this for now but want to circle back to see if you had any updated thoughts on how to handle this.

Reply:
Sure — I’m not sure there’s a super elegant way to do so while continuing to support Node 8, but I can definite dig in a bit more